packages/cli/src/zosfiles/compare/CompareBaseHelper.ts

```diff
@@ -146,7 +146,7 @@ export class CompareBaseHelper {
                 });
             }
             // reading local file as buffer
-            lfContentBuf = fs.readFileSync(localFileHandle);
+            lfContentBuf = IO.readFileSync(localFile, true);
```

Comment:
Is there a reason that we need to normalize the newlines? I am asking because `fs` is going through the process of locking the file and setting the file handle, but if we use IO.readFileSync, we are then opening the file again, which may have changed on disk.

Alternatively, we can enhance IO in Imperative to accept file handles (numbers) and strings, and move forward with that. But that would require an Imperative change and version bump.

Reply:
I don't remember the reason but this fixed an issue and was recommended by Fernando.

Reply:
The reason might have been to address this issue that Trae mentioned where all lines where shown as changed instead of just one: https://github.com/zowe/zowe-cli/pull/1672#pullrequestreview-1417375164

I don't think we need to use `IO.readFileSync` necessarily - perhaps we could call `IO.processNewlines(fs.readFileSync(localFileHandle))` here?

Reply:
If we could do that, it would be appreciated. In the mean time, I might open a PR on Imperative to allow numbers to be passed in.